packages/uswds-core/src/styles/mixins/general/table.scss

```diff
@@ -221,17 +221,22 @@ $table-unsorted-icon-color: get-color-token-from-bg(
   th {
     background-clip: padding-box;
     color: $table-header-text-color;
-    font-weight: fw("bold");
     line-height: line-height(
       $theme-body-font-family,
       $theme-input-line-height
     );
   }
 
-  td,
+  thead td,
+  tfoot td,
   th {
     background-color: color($theme-table-header-background-color);
     color: $table-header-text-color;
+    font-weight: fw("bold");
+  }
```

Comment:
This sets core table header styles, which should apply to all `th`, `thead td`, and `tfoot td`